src/main/daml/Daml/Finance/Holding/Base.daml

```diff
@@ -11,13 +11,13 @@ import Daml.Finance.Interface.Util.Lockable qualified as Lockable (I, Lock(..),
 import Daml.Finance.Util.Disclosure (addObserversImpl, removeObserversImpl, setObserversImpl)
 import Daml.Finance.Util.Lockable (acquireImpl, isValidLock, releaseImpl)
 
--- | Type synonym for `NonFungibleNonTransferable`.
-type T = NonFungibleNonTransferable
+-- | Type synonym for `Base`.
+type T = Base
 
 -- | Implementation of a non-transferable holding.
```

Comment:
I would explain what makes this holding the "base" one

Reply:
-- | Implementation of a base holding, i.e., a non-fungible and non-transferable holding.